.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions:
+  contents: read
+
+jobs:
+  phpunit:
+    name: Run tests PHP ${{ matrix.php }} Symfony ${{ matrix.symfony }} ${{ matrix.dependency }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+      matrix:
+        php:
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0'
+          - '8.1'
+          - '8.2'
+          - '8.3'
+          - '8.4'
+        symfony:
+          - '3.*'
+          - '4.*'
+          - '5.*'
+          - '6.*'
+        dependency:
+          - 'highest'
+        include:
+          - { php: '7.1', symfony: '3.*', dependency: 'lowest' }
+        exclude:
```

Comment:
Let's try removing the `exclude` matrix configuration altogether and see if the tests will fail. Later on, we can bring it back if necessary.